backend/common/src/main/java/ai/verta/modeldb/common/CommonDBUtil.java

```diff
@@ -173,7 +173,8 @@ protected void createTablesLiquibaseMigration(
       DatabaseConfig config,
       String changeSetToRevertUntilTag,
       String liquibaseRootPath,
-      ResourceAccessor resourceAccessor)
+      ResourceAccessor resourceAccessor,
+      boolean newLiquibaseFilenames)
```

Comment:
😭

Reply:
yeah, this is not awesome. Especially as I can't figure out how to know which filename structure a given app is going to use!